hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -466,13 +466,31 @@
    */
   protected String useThisHostnameInstead;
 
+  /**
+   * @deprecated
```

Comment:
Please also add the version when this one was deprecated and in which release it is going to be removed. Check the [HBase book](https://hbase.apache.org/book.html#hbase.versioning.post10) for details.